### ISSUE TITLE
Tokens getting locked

### ISSUE THREAD

**Issue by teorailin** [Just-Msv/Tokens-Gen#2]
All tokens gets locked, Is there any way to fix it?

**Comment by Just-Msv** [comment]
can u show me a screenshot pls

**Comment by teorailin** [comment]
I cant atm, but with locked I mean that it ask for a phone

**Comment by jerlam06** [comment]
Yes, I confirm. The code is working well and all, but discord seems to know there is something weird because every token generated need to be verified with phone number. Any idea why?

**Comment by Just-Msv** [comment]
yep

**Comment by denveloperj** [comment]
I am building an extension for SMSPVA that will get around this. I also believe that they are being flagged because of the tempmail address and not outlook.com or gmail.
